lib/chef/resource/rhsm_register.rb

```diff
@@ -79,6 +79,23 @@ class RhsmRegister < Chef::Resource
         default: false, desired_state: false,
         introduced: "15.9"
 
+      property :server_url, String,
+        description: "The hostname of the subscription service to use. The default is for Customer Portal Subscription Management, subscription.rhn.redhat.com. If this option is not used, the system is registered with Customer Portal Subscription Management.",
+          introduced: "17.8"
+
+      property :base_url, String,
+        description: "The hostname of the content delivery server to use to receive updates. Both Customer Portal Subscription Management and Subscription Asset Manager use Red Hat's hosted content delivery services, with the URL https://cdn.redhat.com. Since Satellite 6 hosts its own content, the URL must be used for systems registered with Satellite 6.",
+        introduced: "17.8"
+
+      property :service_level, String,
+        description: "Sets the service level to use for subscriptions on the registering machine. This is only used with the auto_attach option.",
+        introduced: "17.8"
+
+      property :release,
+        [Integer, Float, String],
+        description: "Sets the operating system minor release to use for subscriptions for the system. Products and updates are limited to the specified minor release version. This is used only used with the auto_attach option.",
```

Comment:
Can we get an example of this in the description sentence?

Reply:
all set!